package com.nit.test.Exception;

public class UserNotFound  extends Exception{
	public UserNotFound() {

		super("UserNotFound ....");

		
	}

}
